src/main/resources/application.properties

```diff
@@ -26,3 +26,15 @@ quarkus.hibernate-orm.database.generation=none
 quarkus.native.additional-build-args =\
     -H:ResourceConfigurationFiles=resources-config.json,\
     -H:ReflectionConfigurationFiles=reflection-config.json
+
+scheduled.survey.publish.interval=10m
+
+mp.messaging.incoming.status.connector=smallrye-mqtt
```

Comment:
Is there a reason behind these variables are reassigned? Tests seem to use full paths (e.g. mp.messaging.incoming.status.host instead of mqtt.host), either use shorthands in both or full setting keys in both

Reply:
Partly resolved.